website/integrations/services/argocd/index.md

```diff
@@ -163,17 +155,15 @@ dex.config: |
       id: authentik
 ```
 
-### Step 3 - Map the `ArgoCD Admins` group to ArgoCD's admin role
-
-In the `argocd-rbac-cm` ConfigMap, add the following to the data field (or create it if it's not already there) :
+### Step 3: Map the ArgoCD Admins group to ArgoCD's admin role
 
+In the `argocd-rbac-cm` ConfigMap, add the following to the `data` field (or create it if it's not already there):
 ```yaml
 policy.csv: |
     g, ArgoCD Admins, role:admin
     g, ArgoCD Viewers, role:readonly
 ```
 
-If you already had an "admin" group and thus didn't create the `ArgoCD Admins` one, just replace `ArgoCD Admins` with your existing group name.
-If you did not opt to create a read-only group, or chose to use one with a different name in authentik, rename or remove here accordingly.
+If you already had an "admin" group and thus didn't create the `ArgoCD Admins` one, just replace `ArgoCD Admins` with your existing group name. If you did not opt to create a read-only group, or chose to use one with a different name in authentik, rename or remove here accordingly.
```

Comment:
If you already had an admin group in authentik? If yes, let's add that clarification.